### ISSUE TITLE
request: can we have live wallpaper of Boo

### ISSUE THREAD

**Issue by MrYogi** [larvalabs/boo#1]
can we have live wallpaper of Boo
that will be amazing!!

**Comment by megamattron** [comment]
A fun idea! After looking into it briefly there doesn't seem to be any restrictions on using the camera in the live wallpaper, so it should be possible...

**Comment by JMVita** [comment]
Good idea. Oh wait, can wee add more creatures than this circle characters? I like this boo app veryy much. Also, try to add modes like they will jumpscare if they see your face like boo! or add a mouth reaction.
